CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 ## [Unreleased]
 
+## [v0.5.0]
+
 ### Added
 
 - [[#8]](https://github.com/rust-vmm/vm-memory/issues/8): Add GuestMemory method to return an Iterator
+- [[#120]](https://github.com/rust-vmm/vm-memory/pull/120): Add is\_hugetlbfs() to GuestMemoryRegion
```

Comment:
What is with the `\` in the middle of the function name? Is that intentional?

Reply:
My editor recommends me to add a "\" here, fixed:)